centralledger/chart-service/values.yaml

```diff
@@ -234,63 +234,7 @@ config:
 ##    command: ['sh', '-c', 'echo "hello world"']
 ##
 # initContainers: []
-initContainers: |
-  - name: wait-for-mysql
```

Comment:
Don't we need to leave `wait-for-mysql` initContainer for CL service  to check if DB is available before string the service?

Reply:
yes, I will restore this